pkgs/shells/fish/plugins/forgit.nix

```
@@ -4,7 +4,13 @@ buildFishPlugin rec {
   pname = "forgit";
   version = "unstable-2021-04-09";
 
-  buildInputs = [ git fzf ];
+  preFixup = ''
+    substituteInPlace $out/share/fish/vendor_conf.d/forgit.plugin.fish \
+      --replace "fzf " "${fzf}/bin/fzf "
+
+    substituteInPlace $out/share/fish/vendor_conf.d/forgit.plugin.fish \
+      --replace "git " "${git}/bin/git "
+  '';
```

You can combine both:

```suggestion
  preFixup = ''
    substituteInPlace "$out/share/fish/vendor_conf.d/forgit.plugin.fish" \
      --replace "fzf " "${fzf}/bin/fzf " \
      --replace "git " "${git}/bin/git "
  '';
```